website/docs/docs/build/snapshots.md

```diff
@@ -83,7 +83,7 @@ The following table outlines the configurations available for snapshots:
 | [schema](/reference/resource-configs/schema) | Specify a custom schema for the snapshot | No | snapshots |
 | [alias](/reference/resource-configs/alias)   | Specify an alias for the snapshot | No | your_custom_snapshot |
 | [strategy](/reference/resource-configs/strategy) | The snapshot strategy to use. Valid values: `timestamp` or `check` | Yes | timestamp |
-| [unique_key](/reference/resource-configs/unique_key) | A <Term id="primary-key" /> column or expression for the record | Yes | id |
+| [unique_key](/reference/resource-configs/unique_key) | A <Term id="primary-key" /> column(s) (string or array) or expression for the record | Yes |  `id` or `[order_id, product_id]` |
```

Comment:
Should I remove "or expression for the record" because we're no longer supporting a combination of two columns?

Reply:
I think SQL expressions are still allowed, so this looks good to me 👍